bitnami/jaeger/values.yaml

```diff
@@ -0,0 +1,1131 @@
+  ## Minimum memory for development is 4GB and 2 CPU cores
+  ## Minimum memory for production is 8GB and 4 CPU cores
+  ## ref: http://docs.datastax.com/en/archived/jaeger/2.0/jaeger/architecture/architecturePlanningHardware_c.html
+  ##
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param migration.resources.limits The resources limits for Jaeger containers
+  ## @param migration.resources.requests The requested resources for Jaeger containers
+  ##
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 2
+    ##    memory: 4Gi
+    ##
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 2
+    ##    memory: 4Gi
+    ##
+    requests: {}
+  ## @param migration.extraVolumes Optionally specify extra list of additional volumes for jaeger container
+  ##
+  extraVolumes: []
+
+## @section Cassandra storage pod
+##
+cassandra:
+  ## @param cassandra.enabled Enables cassandra storage pod
+  ##
+  enabled: true
+  ## @param cassandra.keyspace Name for cassandra's jaeger keyspace
+  ##
+  keyspace: "jaeger_v1_dc1"
+  ## Database credentials
+  ## @param dbUser.user Cassandra admin user
+  ## @param dbUser.password Password for `dbUser.user`. Randomly generated if empty
+  ##
+  dbUser:
+    user: cassandra
+    password: ""
+  ## Cassandra port
+  ## @param cassandra.containerPorts.cql Cassandra cql port
+  containerPorts:
+      cql: 9042
+
+## @param cqlshImage Set the image to use for the migration job
+## @param cqlshImage.registry Cassandra image registry
+## @param cqlshImage.repository Cassandra image repository
+## @param cqlshImage.tag Cassandra image tag (immutable tags are recommended)
+## @param cqlshImage.digest Cassandra image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+## @param cqlshImage.pullPolicy image pull policy
+## @param cqlshImage.pullSecrets Cassandra image pull secrets
+## @param cqlshImage.debug Enable image debug mode
+##
+cqlshImage:
+  registry: docker.io
+  repository: bitnami/cassandra
+  tag: 4.0-debian-11
+  digest: ""
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+
+externalDatabase:
+  host: ""
+  port: 5432
```

Comment:
This is not Cassandra default port, this is PostgreSQL

Reply:
Fixed [bc004d8](https://github.com/bitnami/charts/pull/13480/commits/bc004d85d2836be1975fb2891c3a6074610cc65e)